.travis.yml

```diff
@@ -52,3 +52,9 @@ deploy:
   on:
     tags: true
     condition: $DEPLOY_ARTIFACTS = 1
+
+notifications:
+  slack:
+    rooms:
+      - secure: "4NsPGDKKtiUSdqjl3bxUMlbw"
+    on_success: always
```

Comment:
I think you want `on_failure: always` if this bot's supposed to ping when things break. Also, perhaps add `on_pull_requests: false` to avoid spurious notifications for PRs, and `on_success: change` for when builds change from failing to passing?